optuna/storages/_grpc/server.py

```diff
@@ -456,9 +456,9 @@ def run_grpc_proxy_server(
         .. code::
 
             from optuna.storages import run_grpc_proxy_server
-            from optuna.storages import RDBStorage
+            from optuna.storages import get_storage
 
-            storage = RDBStorage("sqlite:///example.db")
```

Comment:
As described below, there are limitations to use SQLite3 as a backend for gRPC storage proxy. Therefore, this PR changes the database from SQLite3 to MySQL.
https://github.com/optuna/optuna/blob/ec2c40b50e3c0812d7722d3d6c5b1f52ae4cd9bf/optuna/storages/_grpc/client.py#L57-L60